### PR TITLE
[stable/mysql] support imagePullSecrets

### PR DESCRIPTION
**Pull request by wwojcik**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/mysql/values.yaml [hunk 1]
```diff
@@ -28,6 +28,11 @@ imageTag: "5.7.14"
 ##
 imagePullPolicy: IfNotPresent
 
+## Optional array of imagePullSecrets containing private registry credentials
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
```
Comment (mattfarina) [inline stable/mysql/values.yaml]: Can you comment out `imagePullSecrets`? That's the way it's handled elsewhere. Looking for consistency.

For example,
https://github.com/kubernetes/charts/blob/da7d836a29f30c6f5b469686643a72f64dd983e0/incubator/oauth-proxy/values.yaml#L16-L20

Comment (wwojcik) [reply stable/mysql/values.yaml]: Thanks changed

**Pull request merged**